### ISSUE TITLE
NVM installation causes script to fail

### ISSUE THREAD

**Issue by monfresh** [18F/laptop#80]
Here is the log output:

```
Checking on Node.js installation...

Installing nvm and lts Node.js and npm...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 10007  100 10007    0     0  15004      0 --:--:-- --:--:-- --:--:-- 15002
=> Downloading nvm from git to '/Users/moncef/.nvm'
=> Cloning into '/Users/moncef/.nvm'...
* (HEAD detached at v0.31.7)
  master

=> Appending source string to /Users/moncef/.zshrc
=> Close and reopen your terminal to start using nvm or run the following to use it now:

export NVM_DIR="/Users/moncef/.nvm"
[ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh"  # This loads nvm
mac: line 133: nvm: command not found
failed
```

Note that for the script to have access to nvm, those commands need to be run.

cc @msecret

**Comment by monfresh** [comment]
Installing nvm manually is also broken for some reason:
```
nvm install --lts
Installing latest LTS version.
Downloading https://nodejs.org/dist/v4.6.0/node-v4.6.0-darwin-x64.tar.xz...
######################################################################## 100.0%
Checksums do not match.
Binary download failed, trying source.
######################################################################## 100.0%
Checksums empty
Now using node v4.6.0 (npm v2.15.9)
nvm_ensure_default_set: a version is required
```

**Comment by monfresh** [comment]
@msecret @rogeruiz Had you tested this on a machine that did not already have NVM installed?

**Comment by rogeruiz** [comment]
I'm looking into this today @monfresh!

**Comment by monfresh** [comment]
Fixed in #93
